data/paradise world jobs.txt

```diff
@@ -365,3 +365,23 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+
+mission "Premium materials to <planet>"
+	job
+	repeat
+	deadline
+	description "Deliver premium materials for new high end condo complex on <destination> by <date>. Payment is <payment>."
```

Comment:
> for new high end condo complex

for a new high end condo complex
or
for new high end condo complexes

I'd personally say high-end but I don't think that's required

Reply:
> I'd personally say high-end but I don't think that's required

Thanks for pointing that out I'll update it.

Right now it is just one complex not more than one. Although I am not sure of the cargo weight values if they should be increased or decreased.